packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -0,0 +1,49 @@
+import type { ReactElement } from 'react';
+import { useState } from 'react';
+import styled from 'styled-components/native';
+import type { BaseInputProps } from './BaseInput';
+import getBaseInputStyles from './getBaseInputStyles';
+import type { Theme } from '~components/BladeProvider';
+
+export const StyledNativeBaseInput = styled.TextInput(
+  (props: BaseInputProps & { isFocussed: boolean; theme: Theme }) => ({
+    ...getBaseInputStyles({ theme: props.theme }),
+    backgroundColor: props.isFocussed
+      ? props.theme.colors.brand.primary[300]
+      : props.theme.colors.brand.gray[200],
+    borderBottomColor: props.isFocussed
+      ? props.theme.colors.brand.primary[500]
+      : props.theme.colors.brand.gray[400],
+  }),
+);
+
+export const StyledBaseInput = ({
+  label,
+  labelPosition = 'top',
+  placeholder,
+  type = 'text',
+  defaultValue,
+  name,
+  onChange,
+}: BaseInputProps): ReactElement => {
+  const [isFocussed, setIsFocussed] = useState(false);
+  console.log({
+    label,
+    labelPosition,
+    onChange,
+  });
```

Comment:
Stray log? Should get our eslint to flag these

Reply:
yeah will get removed, had to put in logs else unused var error eslint.